google-ads/src/main/java/com/google/ads/googleads/lib/callables/ExceptionTransformingUnaryCallable.java

```diff
@@ -80,7 +79,9 @@ public void onFailure(Throwable throwable) {
       if (throwable instanceof CancellationException && cancelled) {
         // this just circled around, so ignore.
       } else if (throwable instanceof ApiException) {
-        setException(transformation.transform((ApiException) throwable));
+        setException(transformation.transform(throwable));
```

Comment:
(continuing the earlier comment here after the force push)

With this check, do you still need `transformation.transform()` to use `Throwable` in its return type and args?

Reply:
I think this is the right type for this interface: we're given Throwable everywhere by GAX, so from a consistency point of view it seems correct to process throwable here. I agree that in general we want to avoid dealing with throwable.